depthai_helpers/cli_utils.py

```diff
@@ -44,6 +44,20 @@ def parse_args():
     parser.add_argument("-brd", "--board", default=None, type=str,
                         help="BW1097, BW1098OBC - Board type from resources/boards/ (not case-sensitive). "
                              "Or path to a custom .json board config. Mutually exclusive with [-fv -rfv -b -r -w]")
+    parser.add_argument("-sh", "--shaves", default=None, type=int,
+                        help="Number of shaves used by NN.")
+    parser.add_argument("-cmx", "--cmx_slices", default=None, type=int,
+                        help="Number of cmx slices used by NN.")
+    parser.add_argument("-NCE", "--NCEs", default=None, type=int,
```

Comment:
Minor thing, should we rename the short version of this argument `"-NCE"` to lowercase, to be consistent with the other arguments?